src/device.rs

```diff
@@ -55,10 +67,24 @@ pub fn output_device() -> Result<VirtualDevice, Box<dyn Error>> {
     relative_axes.insert(RelativeAxisType::REL_WHEEL);
     relative_axes.insert(RelativeAxisType::REL_MISC);
 
+    let info = AbsInfo::new(0, 0, 25000, 0, 0, 200);
+    let tilt_info = AbsInfo::new(0, 0, 60, 0, 0, 57);
+    let press_info = AbsInfo::new(0, 0, 8191, 0, 0, 0);
```

Comment:
I'm not so sure about how I could maintain those non-zero numbers (`25000`, `200`, `60`, `57`, `8191`). Can you leave a comment that has a link to some reference or use a library-provided constant like the following variables?

Reply:
Sorry for not testing this before, it appears these values decide what area of the tablet is usable an how accurate the movement is. The current values were just a copy paste from my tablet's file descriptor but it would be desirable to make them configurable in the config.